### PR TITLE
Kanister docs update to version 0.96.0

### PR DESCRIPTION
**Pull request by infraq**
Kanister docs update to version 0.96.0

### CODE REVIEW

**Review by pavannd1 — CHANGES_REQUESTED** [review]
Holding off until the end of the week.

**Review by pavannd1 — APPROVED** [review]

**Review by pavannd1 — CHANGES_REQUESTED** [review]
Found a vulnerability in the kanister-tools image. Going to fix it before merging this.

**Review by pavannd1 — APPROVED** [review]

**Comment by pavannd1** [comment]
Merge after #2351

**Review by viveksinghggits — APPROVED** [review]

**Pull request merged**
